api/axios-instance.js

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
 const axiosAdmin = axios.create({
-	baseURL: "",
+	baseURL: "https://vag-server.herokuapp.com/api",
```

Comment:
Let`s use .env variable to pass the baseURL

Reply:
Okay, good idea.